posthog/temporal/batch_exports/postgres_batch_export.py

```diff
@@ -420,7 +421,22 @@ async def copy_tsv_to_postgres(
                     while data := await asyncio.to_thread(tsv_file.read):
                         # \u0000 cannot be present in PostgreSQL's jsonb type, and will cause an error.
                         # See: https://www.postgresql.org/docs/17/datatype-json.html
-                        data = data.replace(b"\\u0000", b"")
+                        # We use a regex to avoid replacing escaped \u0000 (for example, \\u0000, which we have seen in
+                        # some actual data)
+                        data = re.sub(rb"(?<!\\)\\u0000", b"", data)
```

Comment:
Could we compile these regular expressions just in case they happen to land in some hot path?